easybuild/easyconfigs/a/arpack-ng/arpack-ng-3.7.0-intel-2019b.eb

```
@@ -0,0 +1,46 @@
+easyblock = 'CMakeMake'
+
+name = 'arpack-ng'
+version = "3.7.0"
+
+homepage = 'https://github.com/opencollab/arpack-ng'
+description = """ARPACK is a collection of Fortran77 subroutines designed to solve large scale eigenvalue problems."""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+github_account = 'opencollab'
+source_urls = [GITHUB_SOURCE]
+sources = ['%(version)s.tar.gz']
+patches = ['%(name)s-%(version)s-add_include_CheckSymbolExists_CMakeLists_txt.patch']
+checksums = [
+    '972e3fc3cd0b9d6b5a737c9bf6fd07515c0d6549319d4ffb06970e64fa3cc2d6',  # arpack-ng-3.7.0.tar.gz
+    # arpack-ng-3.7.0-add_include_CheckSymbolExists_CMakeLists_txt.patch
+    'bb7aba0fa067018cabe86f361b0a17c03c0b2153d141f5de3e1fbea02891e90f',
+]
+
+builddependencies = [
+    ('CMake', '3.15.3')
+]
+
+dependencies = [
+    ('Eigen', '3.3.7', '', True)
+]
+
+separate_build_dir = True
+
+local_common_configopts = "-DICB=ON -DMPI=ON -DEXAMPLES=ON"
```

I suggest to change the installation directory of the libraries to match the installations of other `arpack-ng` easyconfigs
```suggestion
local_common_configopts = "-DCMAKE_INSTALL_LIBDIR=lib -DICB=ON -DMPI=ON -DEXAMPLES=ON"
```